Dockerfile

```diff
@@ -15,11 +15,12 @@ RUN set -ex \
     && { go get -d github.com/github/orchestrator ; : ; } \
     && cd $GOPATH/src/github.com/github/orchestrator \
     && bash build.sh -b \
-    && cp ./docker/entrypoint.sh /entrypoint.sh \
```

Comment:
The entrypoint is added later in the `Dockerfile` (`ADD docker/entrypoint.sh /entrypoint.sh`) so that if it's changed `docker build` picks up the change and rebuild the the container.